prow/cluster/jobs/istio/community/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
```

Comment:
Is the approve plugin turned on in this repo?

Reply:
You mean on  test-infra?

I'm just  following the pattern of the other jobs, they  have a similar OWNERS file.

Reply:
oh right, I'm dumb